src/http/cookie.cr

```diff
@@ -156,13 +156,13 @@ module HTTP
         end
       end
 
-      def parse_cookies(header)
+      def parse_cookies(header) : Array(HTTP::Cookie)
```

Comment:
Same same, there's multiple occurrences of these so to avoid spamming I've added suggestion to just this instance.

```suggestion
      def parse_cookies(header) : Array(Cookie)
```